ansible_collections/arista/avd/roles/eos_designs/templates/network_services/logic.j2

```diff
@@ -0,0 +1,34 @@
+
+{% set tmp_tenants = [] %}
+{# tenants #}
+{% for tenant in tenants | arista.avd.convert_dicts | arista.avd.natural_sort('name') %}
```

Comment:
Why not loop through switch.tenants, so that we eliminate the next if condition. ``` for tenant in switch.tenants``` and then to access the tenant data we can use tenants[tenant] directly. Similarly for other data like, vrfs and svis. Because we did this similar things already in switch facts.